src/main/flight/mixer.c

```diff
@@ -269,83 +273,123 @@ static void calculateThrottleAndCurrentMotorEndpoints(timeUs_t currentTimeUs)
+        if (stickDeflectionRollAbs > stickDeflectionPitchAbs) {
             signPitch = 0;
         } else {
-            // If pitch/roll dominant, disable yaw
-            signYaw = 0;
+            signRoll = 0;
         }
+    }
+
+    // Calculate crashflipPower from stick deflection with a reasonable amount of stick deadband
+    float crashflipPower = stickDeflectionLength > CRASHFLIP_STICK_MINF ? stickDeflectionLength : 0.0f;
+
+    // calculate flipPower attenuators
+    float crashflipRateAttenuator = 1.0f;
+    float crashflipAttitudeAttenuator = 1.0f;
+    const float crashflipRateLimit = mixerConfig()->crashflip_rate * 10.0f; // eg 35 = no power by 350 deg/s
+    const float smallAttChange = 0.5f; // attenuation factors less that this will be ignored
+    const float smallAttChangeInv = 1.0f / smallAttChange;
+
+    // disable both attenuators if the user's crashflip_rate is zero
+    if (crashflipRateLimit > 0) {
+#ifdef USE_ACC
+        // Calculate an attenuator based on change of attitude (requires Acc)
+        // with Acc, crashflipAttitudeAttenuator will be zero after approx 90 degree rotation, and
+        // motors will stop / not spin while attitude remains more than ~90 degrees from initial attitude
+        // without Acc, the user must manually center the stick, or exit crashflip mode, or disarm, to stop the motors
+        // re-initialisation of crashFlip mode by arm/disarm is required to reset the initial tilt angle
+        if (sensors(SENSOR_ACC)) {
+            const float tiltAngle = getCosTiltAngle();  // -1 if flat inverted, 0 when 90° (on edge), +1 when flat and upright
+            if (isFirstTiltAngleRead) {
+                tiltAngleAtStart = tiltAngle;
+                isFirstTiltAngleRead = false;
+            }
+            const float attitudeChange = fmaxf(1.0f - fabsf(tiltAngle - tiltAngleAtStart), 0.0f);
+            // no attenuation unless a significant attitude change has occurred
+            crashflipAttitudeAttenuator = attitudeChange > smallAttChange ? 1.0f : attitudeChange * smallAttChangeInv;
+        }
+#endif // USE_ACC
+        // Calculate an attenuation factor based on rate of rotation... note:
+        // if driving roll or pitch, quad usually turns on that axis, but if one motor sticks, could be a diagonal rotation
+        // if driving diagonally, the turn could be either roll or pitch
+        // if driving yaw, typically one motor sticks, and the quad yaws a little then flips diagonally
+        float gyroRate = 0.0f;
+        for (int axis = 0; axis < FD_YAW; axis++) {
+            gyroRate = fmaxf(gyroRate, fabsf(gyro.gyroADCf[axis]));
+        }
+        const float gyroRateChange = fmaxf(1.0f - gyroRate / crashflipRateLimit, 0.0f);
+        // no attenuation unless a significant gyro rate change has occurred
+        crashflipRateAttenuator = gyroRateChange > smallAttChange ? 1.0f : gyroRateChange * smallAttChangeInv;
```

Comment:
This deadband is not helpful here because it promotes over-rotation and lowers the ability to fine-tune with `crashflip_rate`, I reckon.

Reply:
but helpful to start semi-blocked motors, no?

Reply:
@KarateBrot - yes, more power for longer = more chance of over-rotation.
But also, more power, strong at the start, and without significant attenuation until 'free', is often what's needed to get out of grass.
With the deadband, we see it working well now on @nerdCopter's low-authority build.
And I did extensive testing with my 5in now, with a setting of 200 deg/s. I have reliable flips, even if I hammer directly to max stick - a sudden motor kick, then over nicely.  If I smoothly move the stick, over about 1s, I get a smoothly controlled turn once I clear the immediate obstruction.  
In contrast, if I set the rate to 1000 deg/s, I get immediate double-flips when hammering the stick.
This clarifies that the deadband doesn't reduce sensitivity to the 'rate' adjustment.
Hence, I think the deadband is quite important for clearing initial obstruction and to overcome small angle/rate changes that otherwise diminish initial power leading to failure to flip, and that it should stay.